### PR TITLE
Remove mention of Katacoda interactive tutorial

### LINKED ISSUE #43301: Some clerical errors in "Learn Kubernetes Basics" Page
In page [“Learn Kubernetes Basics”](https://kubernetes.io/docs/tutorials/kubernetes-basics/), the first part describes that an online interative tutorial is included. But in fact, this tutorial was shutdown a few months ago.

I think that may confuse some readers.

### PR DESCRIPTION
**Pull request by Affan-7**
Fixes #43301

Updated:
docs/tutorials/kubernetes-basics/deploy-app/deploy-intro

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -24,15 +24,7 @@
...
-        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
-        <p>Using the interactive tutorials, you can learn to:</p>
-        <ul>
-        <li>Deploy a containerized application on a cluster.</li>
-        <li>Scale the deployment.</li>
-        <li>Update the containerized application with a new software version.</li>
-        <li>Debug the containerized application.</li>
-        </ul>
-        
+        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features, concepts and a tutorial for you to follow along.</p>
```
Comment (aj11anuj) [inline content/en/docs/tutorials/kubernetes-basics/_index.html:27]:
```suggestion
        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features, concepts and a tutorial for you to follow along. These tutorials will let you manage a simple cluster and its containerized applications for yourself.</p>
        <p>Using these tutorials, you can learn to:</p>
        <ul>
        <li>Deploy a containerized application on a cluster.</li>
        <li>Scale the deployment.</li>
        <li>Update the containerized application with a new software version.</li>
        <li>Debug the containerized application.</li>
        </ul>
        
```

**Review by aj11anuj — COMMENTED** [review]
Users will still be able to learn all these things, just not in an interactive way. I think it isn't required to remove all these points. Removing just "interactive" word will be fine.

File: content/en/docs/tutorials/kubernetes-basics/_index.html (hunk 1 above)
Comment (Affan-7) [reply content/en/docs/tutorials/kubernetes-basics/_index.html:27]: Yes, that makes sense. Let me just change it.

Thanks

**Comment by sftim** [comment]
/retitle Remove mention of Katacoda interactive tutorial

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

This change makes the site less wrong.

**Review by huhusuzyzzz — COMMENTED** [review]
di.qq_1@icloud.com

**Review by huhusuzyzzz — COMMENTED** [review]
Q7141FL9K0

**Pull request merged**